docs/standard/serialization/system-text-json-converters-how-to.md

```diff
@@ -236,6 +236,16 @@ For scenarios that require type inference, the following code shows a custom con
 * Strings to `string`
 * Everything else to `JsonElement`
 
+::: zone pivot="dotnet-5-0"
+
+:::code language="csharp" source="snippets/system-text-json-how-to-5-0/csharp/CustomConverterInferredTypesToObject.cs":::
+
+The example shows the converter code and a `WeatherForecast` class with `object` properties. The `Main` method deserializes a JSON string into a `WeatherForecast` instance, first without using the converter, and then using the converter. The console output shows that without the converter the run time type for the `Date` property is `JsonElement`; with the converter, the run time type is `DateTime`.
+
+:::zone-end
+
+::: zone pivot="dotnet-core-3-1"
+
```

Comment:
Is there any reason for the difference in writing style between the 3.1 and 5.0 pivots? In 3.1, we break up the code sample and our explanation into chunks:  "The following code registers the converter:", "Here's an example with `object` properties:", and so on; whereas in 5.0, there's just one mass of code, followed by some text. 

It's not a critical crucial difference, just something that struck me as I compared the two pivots. We could make them consistent, or leave as-is (up to you).

Reply:
After publishing the 3.1 STJ docs there were some comments complaining about lack of context for the code snippets, so I took a different approach with the snippets created for the 5.0 updates -- each is a complete program that you can copy/paste into a project to run the code. This approach results in longer code examples but if it proves to be more helpful I can gradually change existing snippets created for 3.1 to follow the same pattern. Since the how-to was split up from one big doc into many smaller ones, it's more practical to have bigger code snippets.